rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/activity/CovidPatientRegisterActivity.java

```diff
@@ -29,6 +30,7 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         registerLanguageSwitcher();
+        Utils.verifyUserAuthorization();
```

Comment:
Any reason to move this to `onCreate` instead of `onResume`?  I think the goal is to call this every time the activity enters the foreground after a period of idleness? 

This applies to all activities.

Reply:
done